samples/tsglobal.d.ts

```diff
@@ -1,6 +1,7 @@
 /// <reference path="../code/highcharts.src.d.ts" />
 /// <reference path="../code/dashboard.src.d.ts" />
 /// <reference path="../code/datagrid.src.d.ts" />
+/// <reference path="../code/grid/es-modules/masters-grid/grid-lite.src.d.ts" />
```

Comment:
Maybe unrelated to this PR, but the other paths for datagrid and dashboard need some adjustment as they are missing the subfolders.